api-docs/kotlin/src/main/kotlin/org/bitcoindevkit/bdk.kt

```diff
@@ -578,6 +578,62 @@ class DescriptorPublicKey(network: Network, mnemonic: String, password: String?)
     fun asString(): String
 }
 
+/**
+ * A output descriptor.
+ *
+ * @param descriptor The descriptor in string format.
+ * @param network The network this DescriptorPublicKey is to be used on.
```

Comment:
I think you may have a copy/paste error here, should be Descriptor right?
```suggestion
 * @param network The network this Descriptor is to be used on.
```

Reply:
Oh yep! Thanks for pointing it out.

Reply:
Fixed.